src/libraries/Common/tests/System/Runtime/Serialization/Utils.cs

```diff
@@ -257,6 +257,11 @@ private static bool CompareXElements(XElement baselineXElement, XElement actualX
                             // Check if they both have the same namespace.
                             XNamespace deskns = baselineXElement.GetNamespaceOfPrefix(deskPrefix);
                             XNamespace coreclrns = actualXElement.GetNamespaceOfPrefix(coreCLRPrefix);
+                            if ((deskns == null) || (coreclrns == null))
+                            {
+                                Debug.WriteLine("Either expected {0} or actual {1} attribute value doesn't have namespace :", deskAtrs[i].Value, coreCLRAtrs[i].Value);
```

Comment:
It's possible that they might both not have a namespace. Is this a scenario we care about differentiating in the debug output?